server/src/main/java/org/elasticsearch/repositories/blobstore/BlobStoreRepository.java

```diff
@@ -1352,40 +1351,45 @@ private void cleanupStaleBlobs(
      *     <li>Deleting stale indices</li>
      *     <li>Deleting unreferenced root level blobs</li>
      * </ul>
-     * @param repositoryStateId     Current repository state id
-     * @param repositoryMetaVersion version of the updated repository metadata to write
+     * @param originalRepositoryDataGeneration     Current repository state id
```

Comment:
I'm renaming this because the next step will be to move this method within `SnapshotsDeletion` where we can use the `originalRepositoryDataGeneration` field here. The entry method will go back to `repositoryDataGeneration`